_data/speakers.yml

```diff
@@ -136,6 +139,7 @@
   presentation_type: Let's Build a Model
   time_zone:
   track: Advanced
+  keywork: 
```

Comment:
Everybody must have a keyword, if he takes part in 2 talks he should have the keyword of the "main" talk.

Reply:
Done this, added vincent's keyword to Matthijs